tutorials/learnpython.org/es/Loops.md

```diff
@@ -93,5 +93,5 @@ Expected Output
 126
 328
 
-Solution
---------
\ No newline at end of file
+Solución

```

Comment:
This cannot be translated, please revert to Solution because the code depends on it

Reply:
Ready!